contracts/src/v0.8/shared/canonical-weth-zk/WETH9ZKSync.sol

```diff
@@ -0,0 +1,87 @@
+// Submitted for verification at Etherscan.io on 2017-12-12
```

Comment:
Please put it in an appropriate folder. We have `token/erc20` which should be used, you can create a folder `zksync` inside with this contract.